dogstatsd.py

```diff
@@ -336,11 +339,11 @@ class Server(object):
     """
     A statsd udp server.
     """
-    def __init__(self, metrics_aggregator, host, port, forward_to_host=None, forward_to_port=None):
+    def __init__(self, metrics_aggregator, host, port, forward_to_host=None, forward_to_port=None, buffer_size=DEFAULT_BUFFER_SIZE):
```

Comment:
nitpick: this line is probably too long, you might want to put the last two function arguments in a new line.